lib/mapnik_backend.js

```diff
@@ -309,7 +313,8 @@ MapnikSource.prototype._createMetatileCache = function(options) {
             format: coords[0],
             z: +coords[1],
             x: +coords[2],
-            y: +coords[3]
+            y: +coords[3],
+            metrics: source._mapnik_metrics
```

Comment:
I've accepted this is the pattern to follow here. It's a bit weird for me because if you think about it, you are passing an options object with references to the same instance object: `source._renderMetatile` can access `_uri.query.limits`, but fair enough.

Reply:
I'll change it to `source._uri.query.metrics` to at least follow a similar pattern